src/UseCases/FoodTruck/CreateFoodTruckInputBoundary.java

```diff
@@ -1,9 +1,13 @@
 package UseCases.FoodTruck;
 
+import Entities.Interfaces.IAddon;
 import Entities.Interfaces.IOrderbook;
 import Entities.Menu;
 
+import java.util.HashMap;
+
 public interface CreateFoodTruckInputBoundary {
 
-    Boolean createFoodTruck(String userToken, String name, Menu menu, String status, IOrderbook orderbook, String location);
+    Boolean createFoodTruck(String userToken, String name, Menu menu, String status, HashMap<IAddon, Boolean> addAvail,
```

Comment:
We should refactor the food truck use cases to be shop use cases

Reply:
Noted